public/bitemp_graph.js

```diff
@@ -471,12 +474,20 @@ var barChart = function() {
         })
         .text(function(d) {
           var str = '';
-          setDefaultDispPropBehavior(d);
-          if (displayProperty.indexOf('.') === -1) {
-            str = d.content[displayProperty];
+          if(window.location.href === 'http://localhost:3000/search') { 
```

Comment:
Drop the "http://localhost:3000". If you're accessing it remotely, or if it gets deployed to a different port, this won't work. You could do

    window.location.href.endsWith('/search')